boards/espressif/esp32c3_devkitc/doc/index.rst

```diff
@@ -239,6 +239,6 @@ References
 **********
 
 .. [1] https://www.espressif.com/en/products/socs/esp32-c3
-.. _ESP32C3 Devkitc User Guide: https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32c3/esp32-c3-devkitc-02
+.. _ESP32C3 Devkitc User Guide: https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32c3/ESP32-C3-DevKitC-02
```

Comment:
this still has the "old" name *but* is in fact not referenced anywhere in the page, just like the datasheet and reference manual aren't either so effectively the References section is empty :) 
Might be OK to do in a follow-up PR though, but just thought it would be nice to at least get all the renames right all at once.


![image](https://github.com/user-attachments/assets/1d635507-b1c1-402e-af04-2f3d11fcd838)

Reply:
I'll update this as well.